local_dispatcher/local_dispatch.py

```diff
@@ -19,12 +19,13 @@
 
 
 class Service():
-    def __init__(self, name, host, port, path_function, is_default=False):
+    def __init__(self, name, host, port, path_function, is_default=False, get_service_path=lambda path : path):
```

Comment:
`get_service_path` can be used to modify service path. For example, I use it to remove `proxy_to_minio` portion from the url before sending it to minio.